feat(l2): pay base fee to coinbase (#4536)
lambdaclass/ethrex: docs/l2/running.md

	--committer_l1_private_key <private-key> \
	--proof_coordinator_l1_private_key \
	--block-producer.coinbase-address <l2-coinbase-address> \
	--block-producer.fee-vault-address <l2-fee-vault-address>
```

For further configuration take a look at the [CLI document](../CLI.md#ethrex-l2)

This will start an ethrex l2 sequencer with the RPC server listening at `http://localhost:1729` and the proof coordinator server listening at `http://localhost:3900`

> **Note:** If `--block-producer.fee-vault-address` is set, the sequencer will send collected base fees to that address instead of burning them.  

> > Be cautious: if the fee vault address is the same as the coinbase address, the coinbase balance will change in a way that differs from the standard L1 behavior, which may break assumptions about EVM compatibility.

Author: Nit: the phrasing sounds kind of weird

```suggestion
> ⚠️ Caution: if the fee vault and coinbase addresses are the same, its balance will change in a way that differs from the standard L1 behavior, which may break assumptions about EVM compatibility.
```

Also, we could use a [caution blockquote here](https://github.com/orgs/community/discussions/16925)
